Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -103,10 +103,9 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
 
     self.backgroundGradientView.startPoint = .zero
     self.backgroundGradientView.endPoint = CGPoint(x: 0, y: 1)
-    self.backgroundGradientView.setGradient([
-      (UIColor(white: 0, alpha: 0.5), 0),
-      (UIColor(white: 0, alpha: 0), 1)
-      ])
+    let gradient: [(UIColor?, Float)] =  [(UIColor.init(white: 0.0, alpha: 0.5), 0.0),
```

Comment:
Since we're explicitly declaring `gradient` to be an array of tuples `(UIColor?, Float)` we can simplify
```
[(UIColor.init(white: 0.0, alpha: 0.5), 0.0)
```
to
```
[(UIColor.init(white: 0.0, alpha: 0.5), 0)
```
(unless this is a code style you prefer for the KSR code base)?